doc/Support/Configuration.md

```diff
@@ -140,6 +140,11 @@ snmp.community:
 snmp.max_repeaters: 30
 ```
 
+**CAUTION**: The above example uses the correct, flattened notation whereas you might be tempted to create a
+block for `snmp` with sub-keys `community` and `max_repeaters`.  Do **NOT** do this as the whole `snmp`
+block will be overwritten, replaced with only those two sub-keys.  The config keys in your `seeders` file
```

Comment:
Is this in relation to using Docker? If so I think you need to call that out in this or put this into the docker README.

Reply:
The keys must match those in config_definitions.json

Reply:
This isn't _strictly_ a Docker thing.  The current docs call out both docker and general "automation".  This is true.  You might be deploying LibreNMS in a non-docker environment and still wish to automate day0.

@murrant I'll mention that file.  It's a good call!